using-timescaledb/reading-data.md

```diff
@@ -325,6 +325,37 @@ This query will output data of the form:
  2018-03-09 20:34:40+00 |  1054.10
  2018-03-09 20:53:20+00 |  1037.78
 ```
+
+### Last Observation Carried Forward [](locf)
```

Comment:
I think I might move this to after last point queries? just cause we basically use one in the query. So it might be a nice way to put the two things together. (and explicitly state that in the docs).